src/pages/token/rewards/StakingPanel.tsx

```diff
@@ -120,9 +122,11 @@ export const StakingPanel = ({ className }: { className?: string }) => {
                   key: STRING_KEYS.STAKED,
                 })}
               </WithTooltip>
-              <Tag>
-                {stringGetter({ key: STRING_KEYS.EST_APR, params: { PERCENTAGE: stakingApr } })}
-              </Tag>
+              {stakingApr && (
+                <Tag>
+                  {stringGetter({ key: STRING_KEYS.EST_APR, params: { PERCENTAGE: stakingApr } })}
```

Comment:
oops this file was supposed to go in my last PR; was just fixing an incorrect merge resolution. Will merge the two together anyways

Reply:
does the base pr just need to be rebased/marged with main?

Reply:
ah nah, I'll just merge this into the base, and then the base into main all together!